### ISSUE TITLE
download of all archaeal sequences from genbank fails

### ISSUE THREAD

**Issue by johanneswerner** [kblin/ncbi-genome-download#88]
I am trying to download all archaeal genomes with any assembly status with the following command. 

```bash
ncbi-genome-download -s genbank -F fasta -l all -R all -H -r 5 archaea
Traceback (most recent call last):
  File "/data/miniconda3/envs/ncbi-genome-download/bin/ncbi-genome-download", line 11, in <module>
    sys.exit(main())
  File "/data/miniconda3/envs/ncbi-genome-download/lib/python2.7/site-packages/ncbi_genome_download/__main__.py", line 24, in main
    ret = args_download(args)
  File "/data/miniconda3/envs/ncbi-genome-download/lib/python2.7/site-packages/ncbi_genome_download/core.py", line 151, in args_download
    return config_download(config)
  File "/data/miniconda3/envs/ncbi-genome-download/lib/python2.7/site-packages/ncbi_genome_download/core.py", line 184, in config_download
    download_jobs.extend(create_downloadjob(entry, group, config))
  File "/data/miniconda3/envs/ncbi-genome-download/lib/python2.7/site-packages/ncbi_genome_download/core.py", line 343, in create_downloadjob
    symlink_path = create_readable_dir(entry, config.section, domain, config.output)
  File "/data/miniconda3/envs/ncbi-genome-download/lib/python2.7/site-packages/ncbi_genome_download/core.py", line 387, in create_readable_dir
    get_species_label(entry),
  File "/data/miniconda3/envs/ncbi-genome-download/lib/python2.7/site-packages/ncbi_genome_download/core.py", line 560, in get_species_label
    return entry['organism_name'].split(' ')[1]
IndexError: list index out of range
```

If I set `-s refseq`, it works fine (but with less downloaded organisms). Any idea what is wrong here? I am using version 0.2.9. Any help would be highly appreciated. 

Thank you!

**Comment by kblin** [comment]
There's probably a funky organism name that's breaking my parsing that doesn't exist in RefSeq, let me check.

**Comment by kblin** [comment]
Yes, looks like there's a bunch of entries with no genus, species or strain information whatsoever breaking the parser.

**Comment by johanneswerner** [comment]
Thanks for the update. That was by far the fastest reply I ever got to a newly opened issue. :-)

**Comment by kblin** [comment]
And of course this only triggers if you want human-readable output, because otherwise `ncbi-genome-download` doesn't try to read that field.

**Comment by kblin** [comment]
Whoops, I just realised that #87 by @andrewsanchez would have fixed that in March if I had gotten around to merge that pull request.

**Comment by kblin** [comment]
Fixed in the 0.2.10 release.

**Comment by johanneswerner** [comment]
Thank you very much!
